lib/task/import/csvAuthorityRecordRelationImportTask.class.php

```diff
@@ -235,6 +239,25 @@ private function getRelation($sourceActorId, $targetActorId, $relationTypeId)
     return QubitPdo::fetchColumn($sql, $paramsVariant);
   }
 
+  /**
+   * Use SQL to fetch relations
+   *
+   * @param int $sourceActorId  ID of an actor that possibly relates to others
+   * @param int $targetActorId  ID of an actor that possibly relates to others
+   *
+   * @return array  array of relation IDs
+   */
+  private function getRelations($sourceActorId, $targetActorId)
+  {
+    $sql = "SELECT id FROM relation
+      WHERE subject_id = :subject_id
+      AND object_id = :object_id";
+
+    $results = QubitPdo::fetchAll($sql, [':subject_id' => $sourceActorId, ':object_id' => $targetActorId], array('fetchMode' => PDO::FETCH_ASSOC));
```

Comment:
Okay, I feel bad now but I have another nitpick: the line is too long. 😊   Could you please split the method parameters into separate lines. 🙏